File: ObjectProgrammingPython/Rosliny/mlecz.py
import random

from Rosliny.roslina import Roslina


class Mlecz(Roslina):
    def __init__(self, Swiat, x, y, czyZyje):
        super(Mlecz, self).__init__(Swiat, x, y, czyZyje)
        self.sila = 0
        self.inicjatywa = 0
        self.kolor = "lightblue"
        self.oznaczenie = "m"
        self.czyMozeRozmnazac = 0
        self.wiek = 0

    def akcja(self):
        self.przygotujOrganizm()
        for i in range(3):
            losuj = random.randint(0,100)
            if losuj > 96:
                super().akcja()

    def Wypisz(self):
        return "Mlecz" + self.koncowka()

File: ObjectProgrammingJava/projekt po - 2/src/projekt/rosliny/Mlecz.java
package projekt.rosliny;

import projekt.Wypisanie;
import projekt.swiat.Swiat;

import java.awt.*;
import java.util.Random;

public class Mlecz extends Roslina implements Wypisanie {
    public Mlecz(Swiat swiat, int x, int y, boolean czyZyje){
        polozenieX = x;
        polozenieY = y;
        sila = 0;
        inicjatywa = 0;
        kolor = new Color(170, 227, 79);
        oznaczenie = 'm';
        this.swiat = swiat;
        this.czyZyje = czyZyje;
        this.czyMozeRozmnazac = 0;
        wiek = 0;
    }

    @Override
    public void akcja() {
        przygotujOrganizm();
        Random random = new Random();
        for(int i = 0; i<3; i++) {
            int losuj = random.nextInt(100);
            if (losuj > 96) {
                super.akcja();
            }
        }
    }

    @Override
    public String Wypisz() {
        return "Mlecz" + getKoncowka();
    }
}
